src/util.cpp

```diff
@@ -771,6 +771,11 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
 
     // set window title
     setWindowTitle ( tr ( "About %1" ).arg ( APP_NAME ) );
+
+#    if defined( Q_OS_IOS )
```

Comment:
Why not for all mobile platforms?

Reply:
I cannot test on Android. That's the main reason.

Reply:
OK, I'd go for it and say it needs testing on Android.

Reply:
Done.